src/docs/quick-start/create-plugin-wrappers/plugin-manifest.md

```diff
@@ -0,0 +1,53 @@
+---
+id: plugin-manifest
+title: 'Configure a Plugin Manifest'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Each plugin wrapper is orchestrated by a Plugin Manifest `polywrap.plugin.yaml` file. 
+The Plugin Manfiest provides the Polywrap CLI with basic information it needs to [`build`](../../reference/cli/commands/build) the plugin. 
+A plugin does not need a Polywrap Manifest; the simpler Plugin Manifest is used in its place.
+
+## Content
+
+The Plugin Manifest contains the name of the plugin, programming language used to write the plugin, 
+the location of the plugin's entry file (i.e. the file that exports the plugin contents),
+and the GraphQl schema that declares the plugin's interface. 
+It can optionally contain an array of [URI Redirects](../../concepts/understanding-uri-redirects).
+
+<Tabs
+defaultValue="schema"
+values={[
+{label: 'Schema', value: 'schema'},
+{label: 'Example', value: 'example'},
+]}>
+<TabItem value="schema">
+
+```yaml
+format: # The manifest format version
+name: # (Optional) Name of plugin
+language: # Plugin programming language
+module: # Entry file 
+schema: # GraphQL schema
```

Comment:
Should we remove the GraphQL wording from our docs? :thinking: Not sure which path we should take here tbh

Reply:
I changed it to "Wrapper schema"